packages/dx-react-grid/docs/reference/row-detail-state.md

```diff
@@ -27,4 +27,4 @@ none
 Name | Plugin | Type | Description
 -----|--------|------|------------
 toggleDetailRowExpanded | Action | ({ rowId }) => void | Expands/collapses the specified row.
-expandedRowIds | Getter | Array&lt;number &#124; string&gt; | Currently expanded rows.
+expandedDetailRowIds | Getter | Array&lt;number &#124; string&gt; | Currently expanded rows.
```

Comment:
> Currently expanded rows.

Currently expanded row ids ?

Reply:
I'd prefer not to change the description. As it will lead to full recheck for several days.

Reply:
Ok)